TweetMate/src/main/java/com/ybsystem/tweetmate/models/entities/twitter/TwitterStatus.java

```diff
@@ -71,11 +74,18 @@
     private String convertedRelativeTime;
     private String convertedAbsoluteTime;
     private String convertedVia;
+    private ArrayList<String> imageUrls;
     private boolean isMyTweet;
     private boolean isPublic;
     private boolean isTalk;
     private boolean isDetail;
-    private boolean isMediaTweet;
+    private boolean isTwiMedia;
+    private boolean isThirdMedia;
+    private boolean isVideo;
+
+    // Media
+    private static final Pattern PATTERN_YOUTUBE =
+            Pattern.compile("^https?://(?:www\\.youtube\\.com/watch\\?.*v=|youtu\\.be/)([\\w-]+)");
 
     // Not use
     // private Place place;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

Reply:
There is a reason for this.